Purge "will" from Client Cfg (#663)
uyuni-project/uyuni-docs: modules/client-configuration/pages/cobbler.adoc

To change the organization, log in as an Administrator of the required organization, and re-enable the feature.
====

When the feature has been enabled, any bare metal system connected to the server network is automatically added to the organization when it is powered on.

> The process can take a few minutes, and the system automatically shutd down when it is complete.

Reviewer: ```suggestion
The process can take a few minutes, and the system automatically shuts down when it is complete.
```

=== Provision Bare Metal Systems

Provisioning bare metal systems is similar to provisioning other systems, and can be done using the [guilabel]``Provisioning`` tab.

> However, you can not schedule provisioning, it happens automatically as soon as the system is configured and powered on.

Reviewer: ```suggestion
However, you cannot schedule provisioning, it happens automatically as soon as the system is configured and powered on.
```
